corehq/apps/reminders/forms.py

```diff
@@ -128,8 +128,8 @@ def clean(self, value):
 
 
 class KeywordForm(Form):
-    _cchq_domain = None
-    _sk_id = None
```

Comment:
I would have interpreted that as `simon kelly id`

Reply:
I hope @snopoke is flattered.